docs/source/endpoints/userschema.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This file needs to be included in `endpoints/index.md` as a `toctree` entry after `users`.

Here is its current preview:
https://deploy-preview-707--plone-restapi.netlify.app/endpoints/userschema.html

Reply:
This one was missed. Please do this, and then I approve. Thank you!